src/facet/data/partition/_partition.py

```diff
@@ -141,10 +142,10 @@ def __init__(
         if (
             lower_bound is not None
             and upper_bound is not None
-            and lower_bound >= upper_bound
+            and lower_bound > upper_bound
         ):
             raise ValueError(
-                f"arg lower_bound >= arg upper_bound: [{lower_bound}, {upper_bound})"
+                f"arg lower_bound > arg upper_bound: [{lower_bound}, {upper_bound})"
```

Comment:
What's the reason we don' permit the upper and lower bound to take the same value anymore? Perhaps add this to the release notes?